src/Windows/Avalonia.Win32/Win32Platform.cs

```diff
@@ -200,7 +211,9 @@ public IPopupImpl CreatePopup()
 
         public IRenderer CreateRenderer(IRenderRoot root, IRenderLoop renderLoop)
```

Comment:
This should be moved to `ITopLevelImpl` since we can mix multiple toplevel implementations in the same app (e. g. WPF embedded avalonia control showing avalonia-backed `Popup`)